sr_description/hand/xacro/finger/biotac/biotac.urdf.xacro

```diff
@@ -154,20 +154,20 @@ xmlns:geom="http://playerstage.sourceforge.net/gazebo/xmlschema/#geom">
         <inertia ixx="0.0" ixy="0.0" ixz="0.0" iyy="0.0" iyz="0.0"
         izz="0.0" />
       </inertial>
-      <visual>
+      <!--visual>
```

Comment:
That is a bug in FCL or so, there is only a visual that should not be considered as a collision. So you worked around by making the tip follow the distal and hence make the collision pair be ignored as adjacent links ?